projects/packages/forms/src/blocks/field-hidden/index.js

```diff
@@ -0,0 +1,41 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { unseen } from '@wordpress/icons';
+import defaultSettings from '../shared/settings';
+import { getIconColor } from '../shared/util/block-icons';
+import edit from './edit';
+import save from './save';
+import variations from './variations';
+
+const name = 'field-hidden';
+const settings = {
+	...defaultSettings,
+	title: __( 'Hidden field', 'jetpack-forms' ),
+	description: __(
+		'Invisible to users, this field lets you store extra data with each form submission.',
+		'jetpack-forms'
+	),
+	icon: {
+		foreground: getIconColor(),
+		src: <Icon icon={ unseen } />,
```

Comment:
Not sure but I think this would work:

```suggestion
		src: unseen,
```